app/cypress/integration/search-tables.spec.js

```diff
@@ -97,3 +101,127 @@ describe('When filtering applications', () => {
+  });
+
+  it('can be filtered by a numeric search', () => {
+    cy.mockLogin('analyst');
+    cy.visit('/analyst/organisation-requests');
+    cy.get('#page-content');
+    cy.get('table.search-table > tbody').find('tr').should('have.length', 5);
+    cy.get('thead > tr > td:nth-child(1) > input.form-control')
+      .clear()
+      .type('2');
+    cy.contains('Apply').click();
+    cy.get('#page-content');
+    cy.url().should(
+      'include',
+      '/analyst/organisation-requests?relayVars=%7B%22user_id%22%3A2%7D'
+    );
+    cy.get('table.search-table > tbody').find('tr').should('have.length', 2);
+    cy.get('tbody > tr:nth-child(1) > td:nth-child(1)').should(
+      'have.text',
+      '2'
+    );
+    cy.get('body').happoScreenshot({
+      component: 'Organisation Requests List',
+      variant: 'Filtered By ID'
+    });
+    cy.contains('Clear').click();
+    cy.get('#page-content');
+    cy.get('table.search-table > tbody').find('tr').should('have.length', 5);
+  });
+
+  it('can be filtered by a text search', () => {
+    cy.mockLogin('analyst');
+    cy.visit('/analyst/organisation-requests');
+    cy.get('#page-content');
+    cy.get('table.search-table > tbody').find('tr').should('have.length', 5);
+    cy.get('thead > tr > td:nth-child(4) > input.form-control')
+      .clear()
+      .type('Doug');
+    cy.contains('Apply').click();
+    cy.get('#page-content');
+    cy.url().should(
+      'include',
+      '/analyst/organisation-requests?relayVars=%7B%22email_address%22%3A%22Doug%22%7D'
+    );
+    cy.get('table.search-table > tbody').find('tr').should('have.length', 1);
+    cy.get('tbody > tr:nth-child(1) > td:nth-child(4)').should(
+      'have.text',
+      'Douglas.Fir@hhry.xxx'
+    );
+    cy.get('body').happoScreenshot({
+      component: 'Organisation Requests List',
+      variant: 'Filtered By Email'
+    });
+    cy.contains('Clear').click();
+    cy.get('#page-content');
+  });
+
+  it('can be filtered by an enum search', () => {
+    cy.mockLogin('analyst');
+    cy.visit('/analyst/organisation-requests');
+    cy.get('#page-content');
+    cy.get('table.search-table > tbody').find('tr').should('have.length', 5);
+    cy.get('thead > tr > td:nth-child(6) > select.form-control').select(
+      'Pending'
+    );
+    cy.contains('Apply').click();
+    cy.get('#page-content');
+    cy.url().should(
+      'include',
+      '/analyst/organisation-requests?relayVars=%7B%22status%22%3A%22PENDING%22%7D'
+    );
+    cy.get('table.search-table > tbody').find('tr').should('have.length', 1);
+    cy.get('tbody > tr:nth-child(1) > td:nth-child(5)').should(
+      'have.text',
+      'test_organisation 3'
+    );
+    cy.get('body').happoScreenshot({
+      component: 'Organisation Requests List',
+      variant: 'Filtered By Status'
+    });
+    cy.contains('Clear').click();
+    cy.get('#page-content');
+  });
+
+  it('The application can be filtered by multiple search fields', () => {
```

Comment:
I would suggest that we don't need to duplicate ALL the tests from the ApplicationList tests above. It really just tests all the Filterable components over and over. I think since all filtering types are tested above for the ApplicationList that for ciip_user_organisation, facility & product filtering we only really need this single "multiple search" test to ensure the filtering works for each page.